core/modules/taxonomy/tests/taxonomy.test

```diff
@@ -549,14 +543,6 @@ class TaxonomyTermTestCase extends TaxonomyWebTestCase {
     // free-tagging field created by the default profile.
     $edit[$instance['field_name'] . "[$langcode]"] = backdrop_implode_tags($terms);
 
-    // Preview and verify the terms appear but are not created.
-    $this->backdropPost('node/add/page', $edit, t('Preview'));
-    foreach ($terms as $term) {
-      $this->assertText($term, 'The term appears on the node preview.');
-    }
-    $tree = taxonomy_get_tree($this->vocabulary->machine_name);
-    $this->assertTrue(empty($tree), 'The terms are not created on preview.');
-
```

Comment:
>  A test that adds terms to a new node, then uploads an image, and then checks that the terms have not been created?

Yep, that's exactly what should be tested. This test exists to make sure that terms are only created on node-save, not when doing any other operations on the node form.

Though upon looking at our code, I can see how that accident may have happened in the past with hook_node() (which used to have a "preview" $op), but now with fields... there's no direct integration with the node form at all, it's all in field hooks. Terms specifically are created in `taxonomy_field_presave()`, which isn't ever going to get called on a node preview. So if this should be tested, it should be tested that Field doesn't call any hooks on preview. So I agree this test is out of place and we don't need a replacement.